clients/client-go/codegenerator/model/model.go

```diff
@@ -243,6 +243,7 @@ func (apiDefs APIDefinitions) GenerateCode(goOutputDir string) {
 
 // This package was generated from the schema defined at
 // ` + apiDefs[i].URL + `
+
```

Comment:
how could we make sure this newline wouldn't be removed in future by someone who would notice this extra blank line? 
Maybe worth commenting it in the code that this is intended?